lib/client.js

```diff
@@ -227,7 +234,7 @@ function Client() {
     if (!self.key && !self.token) {
       return cb(new Error("You need to provide either a token or a key."));
     }
-
+    call.json = true
```

Comment:
Always pass `content-type: application/json` as header field in every request. Makes sense.